paper/paper.md

```diff
@@ -0,0 +1,323 @@
+R for plotting lesson.
+The lesson content for subsequent modules is then intermittently committed and
+pushed to GitHub.
+The written report combines all of the skills learned previously to write a
+report that one could share with the United Nations.
+Then, the learners have a chance to put everything they have learned into
+practice by working in small groups on practice problems that cover the course
+content.
+Finally, we end with a short module recapping what the workshop covered and how
+learners can continue to get help and learn more after the workshop ends.
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+Our modules and instructor suggestions are developed in the style of [Software Carpentry](https://software-carpentry.org/):
+
+1. Each module contains learning objectives at the beginning of each lesson and
+a summary of key points.
+1. The five core modules (2 to 6) are designed to be taught via live coding of
+the content to learners.
+This is a central feature of Carpentries lessons, and we believe it is a great
+way to learn how to program because it requires learners to follow along and
+encounter errors that they must debug along the way, fostering additional
+questions about the course content.
+1. We incorporate formative assessments in the form of short practice exercises
+throughout each lesson so that learners can practice what they have learned and
+instructors can gauge learner understanding of the material.
+
+We also incorporate a few additional key components into the curriculum:
+
+1. Each lesson builds off of others, with the goal of creating a final report
+that can be shared with others.
+1. We structured the curriculum such that it could be taught through an
+in-person or virtual workshop.
+1. We require learners to install all software before the workshop (as The
+Carpentries also requires), and ask them to run an example script that tests
+whether everything is installed correctly.
+Learners must send screenshots of the script output to the workshop lead in
+advance in order to attend. This insures that any installation issues can be
+addressed before the workshop begins.
+1. An extensive small group practice module towards the end of the workshop
+allows learners to more independently practice the skills they have learned
+throughout.
+1. The workshop concludes with a recap of what was covered and resources
+available for learners to continue learning and getting help as their skills
+develop.
+
+## Pilot Workshop
+<!-- experience of use -->
+
+We piloted our curriculum during a virtual two-day Software Carpentry workshop.
+In line with The Carpentries style, we had four instructors and six helpers at
+the workshop to assist with learner questions and technical issues.
+We also used the "sticky note" system where learners can indicate their progress
+on exercises and requests for help by using different colored sticky notes.
```

Comment:
So exactly how many different sticky note colors did you use and what did they mean? How did those map to specific zoom reactions? It might be helpful to provide a bit more guidance here. Or is there a reference where the "sticky note" system is described in detail?